src/pages/ServerRules.js

```diff
@@ -83,19 +87,18 @@ class ServerRules extends Component {
 							<li>
 								Do not try to harm your team. This includes game throwing
 								(intentionally losing a game) and team griefing (i.e. breaking
-								blocks under teammates, stealing from chests, blowing up your
-								own side and preventing your team from achieving objectives).
+								blocks under teammates, stealing from chests, destroying your
+								team's side and preventing your team from achieving objectives).
 							</li>
 							<li>
-								Map/stats abuse such as grinding alt accounts, spawn killing,
-								leaving the game right before getting killed, switching teams
-								right before you lose, and anything similar that may positively
-								affect your stats above other players on the server is not
-								allowed.
+								Grinding alt accounts, spawn killing, leaving the game right before getting
+								killed, switching teams right before you lose and anything similar
+								that may unfairly affect your stats is
+								not allowed. This is called map/stat abuse.
```

Comment:
I kinda preferred how this was phased before. Saying "This is called map/stat abuse." doesn't sound the best in my opinion.

Reply:
reverted back to its original state